projects/vdk-control-cli/src/vdk/internal/control/utils/output_printer.py

```diff
@@ -109,16 +109,17 @@ def print_dict(self, data: Dict[str, Any]) -> None:
 
 
 class InMemoryTextPrinter(Printer):
-    output_buffer = io.StringIO()
+    def __init__(self):
+        self.output_buffer = io.StringIO()
```

Comment:
Make this private : https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard#clarification-on-access-modifiers-in-python